src/main/java/us/dot/its/jpo/ode/aws/depositor/AwsDepositor.java

```diff
@@ -158,14 +163,24 @@ public void run(String[] args) throws Exception {
 		boolean depositToS3 = false;
 		AmazonS3 s3 = null;
 		AmazonKinesisFirehoseAsync firehose = null;
-		if (destination != null && destination.equals("s3")) {
-			depositToS3 = true;
-			s3 = createS3Client(awsRegion);
-
-		} else {
-			firehose = buildFirehoseClient(awsRegion);
+		Storage gcsStorage = null;
+
+		if (destination != null) {
+			if (destination.equals("s3")) {
+				depositToS3 = true;
+				s3 = createS3Client(awsRegion);
+	
+			} else if (destination.equals("firehose")) {
+				firehose = buildFirehoseClient(awsRegion);
+			} else if (destination.equals("gcs")) {
+				gcsStorage = StorageOptions.getDefaultInstance().getService();
+			} else {
+				logger.error("Invalid destination: " + destination);
+				System.exit(1);
+			}
 		}
```

Comment:
What happens if the destination is null? Should we add a log statement to inform developers of this case?

Reply:
Sounds good!